client/assets/images/jetpack/vaultpress-logo.svg

```diff
@@ -0,0 +1 @@
1-.83l.09-.1.09-.1.28-.3.09-.1.09-.1c.25-.28.5-.57.74-.86l.08-.1.09-.1c.09-.1.17-.21.25-.31l.08-.1a.757.757 0 00.08-.11l.69-.88.07-.1.08-.11.23-.31.08-.11.08-.1.62-.9.08-.1.07-.11.22-.32.07-.11.07-.11c.19-.3.39-.6.58-.91l.07-.11.06-.11.2-.32.07-.11a.412.412 0 01.07-.11l.54-.93.06-.11.06-.11c.07-.11.13-.22.19-.33l.06-.11.06-.11c.18-.32.35-.63.51-.94l.06-.11.06-.11c.05-.116.107-.23.17-.34l.06-.12c.018-.036.035-.073.05-.11.16-.32.32-.63.47-.95l.06-.11.05-.11.17-.34.05-.12a.369.369 0 00.05-.11c.15-.32.3-.65.45-1a.615.615 0 01.05-.12.546.546 0 010-.11c.05-.12.09-.23.15-.35a.543.543 0 01.05-.11.617.617 0 010-.12c.14-.32.27-.65.41-1v-.12a.369.369 0 000-.11c.05-.12.1-.23.14-.35l.05-.12c.019-.036.035-.073.05-.11.13-.33.26-.66.38-1v-.12l.05-.12c0-.11.08-.23.13-.35v-.12a.482.482 0 000-.11c.127-.34.247-.673.36-1v-.12a.56.56 0 000-.12l.12-.36a.482.482 0 000-.11v-.12c.12-.35.23-.68.33-1v-.12a.56.56 0 010-.12c0-.12.07-.24.11-.36v-.12a.56.56 0 000-.12c.11-.35.21-.69.3-1V12c0-.12.07-.25.1-.36a.56.56 0 000-.12v-.12l.27-1v-.25c0-.13.07-.25.09-.36v-.25c.13-.53.19-.83.19-.83L17.51 3.42 2.42 8.73zm15.73 26.61l-.1.09c-.1.1-.2.2-.31.29l-.1.1-.11.09V35l.74.26-.12.08zm1.48-1.53l-.1.1-.28.3-.09.1-.09.1-1.56-.55v-.92l2.21.77-.09.1zM21 32.24l-.09.1a3.739 3.739 0 01-.25.31l-.08.1-.09.11-2.95-1v-.93l3.54 1.25-.08.06zm10.18-20.93v.12c0 .11-.07.24-.1.36a.56.56 0 000 .12L31 12l-3-1-.39 1 3 1.07v.24c0 .04-.07.23-.11.36V13.9l-3.09-1.08-.39 1 3.15 1.11-.001-.001-.08.24-.12.36a.829.829 0 000 .12v.11l-3.19-1.12-.39 1 3.22 1.13v.24c0 .04-.09.23-.13.35l-.05.12v.11l-4.09-1.44a1.391 1.391 0 00-.19.34l-.06.15-.2.47 4.15 1.47v.11l-.05.12c0 .11-.09.23-.14.35a1.196 1.196 0 00-.05.11c.005.04.005.081 0 .12l-4.15-1.46-.39 1L28 20.28a.369.369 0 01-.05.11.606.606 0 01-.05.12l-.15.34a.606.606 0 01-.05.12l-.05.11-4.11-1.45-.4 1L27.22 22l-.06.12a.537.537 0 01-.05.11l-.16.34-.05.11-.06.12-4-1.42-.39 1 3.94 1.39-.06.11-.06.11c-.05.11-.11.23-.17.34l-.06.11-.06.11-3.88-1.34-.4 1 3.75 1.32-.06.11-.06.11c-.06.11-.12.22-.19.33l-.06.11-.06.11-3.65-1.3-.37 1 3.49 1.23-.06.11-.07.11c-.07.11-.13.22-.2.32l-.06.11-.1.12-3.36-1.18-.4 1 3.17 1.11-.07.11-.07.11-.22.32-.07.11-.08.1-3-1.05-.34.84h-.45l3.15 1.11-.07.1-.08.11-.23.31a.757.757 0 00-.08.11l-.08.1-4.23-1.49v-.35h-2.05l-5.22-12.91-.06-.15s-.33-.84-1-.84H8.8l-1.87-4.71h5.57l5 12.59.65-1.62-.65-.23v-.93l1 .34.39-1-1.36-.48v-.92l1.68.59.39-1-2.07-.73v-.93l2.4.84.39-1-2.79-1v-.93l3.11 1.09.39-1-3.5-1.23v-.93l3.82 1.35.4-1-4.22-1.48v-.8l4.54 1.6.39-1-4.95-1.72v-.92l7 2.45h3.17L17.51 7.29v-.93l13.71 4.82a.502.502 0 00-.04.13zm.49-1.88v.13c0 .107-.03.227-.09.36v.24l-14-4.91v-.93l14.19 5-.1.11z" fill="#AF1E23"/><path d="M33.4 6.45L18.31 1.14a2.32 2.32 0 00-1.6 0L1.62 6.45A2.42 2.42 0 00.05 9.21C.2 10 4 28.3 15.91 38.77a2.4 2.4 0 003.2 0C31 28.3 34.82 10 35 9.21a2.42 2.42 0 00-1.6-2.76zM34.11 9c-.16.76-3.88 18.8-15.59 29.08a1.57 1.57 0 01-2 0C4.79 27.83 1.07 9.79.91 9a1.54 1.54 0 011-1.75L17 2a1.44 1.44 0 011 0l15.11 5.28a1.55 1.55 0 011 1.72z" fill="#AF1E23"/></svg>
```

Comment:
This new VaultPress logo doesn't have any of its own padding, so we can line it up better with the other icons using CSS.